cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/worker/ConfiguratorTask.java

```diff
@@ -86,24 +87,30 @@ public void run(RunnableTaskContext context) throws Exception {
     private final PluginFinder pluginFinder;
     private final ArtifactRepository artifactRepository;
     private final CConfiguration cConf;
+    private final ArtifactLocalizerClient artifactLocalizerClient;
 
     @Inject
     ConfiguratorTaskRunner(Impersonator impersonator, PluginFinder pluginFinder,
-                           ArtifactRepository artifactRepository, CConfiguration cConf) {
+                           ArtifactRepository artifactRepository, CConfiguration cConf,
+                           ArtifactLocalizerClient artifactLocalizerClient) {
       this.impersonator = impersonator;
       this.pluginFinder = pluginFinder;
       this.artifactRepository = artifactRepository;
       this.cConf = cConf;
+      this.artifactLocalizerClient = artifactLocalizerClient;
     }
 
     public ConfigResponse configure(AppDeploymentInfo info) throws Exception {
       // Getting the pipeline app from appfabric
       LOG.debug("Fetching artifact '{}' from app-fabric to create artifact class loader.", info.getArtifactId());
 
-      Location artifactLocation = artifactRepository
-        .getArtifact(Id.Artifact.fromEntityId(info.getArtifactId()))
-        .getDescriptor()
-        .getLocation();
+      //      Location artifactLocation = artifactRepository
```

Comment:
Please remove commented code before merging

Reply:
Will do, just testing out a new approach for now